packages/amplication-server/src/core/git/git.resolver.ts

```diff
@@ -16,21 +21,82 @@ export class GitResolver {
   constructor(private readonly gitService: GitService) {}
   //#region Mutation
   @Mutation(() => GitRepo)
-  @AuthorizeContext(AuthorizableResourceParameter.AppId, 'appId')
+  @AuthorizeContext(
+    AuthorizableResourceParameter.GitOrganizationId,
+    'gitOrganizationId'
+  )
   async createRepoInOrg(@Args() args: CreateRepoArgs): Promise<GitRepo> {
     return this.gitService.createRepo(args);
   }
+
+  @Mutation(() => GitOrganization)
+  @AuthorizeContext(
+    AuthorizableResourceParameter.GitOrganizationId,
+    'gitOrganizationId'
+  )
+  async getOrganization(@Args() args: BaseGitArgs): Promise<GitOrganization> {
+    return this.gitService.getGitOrganization(args);
+  }
+
+  @Mutation(() => GitOrganization)
+  //@AuthorizeContext(AuthorizableResourceParameter.WorkspaceId, 'workspaceId')
+  async createOrganization(
+    @Args() args: CreateGitOrganizationArgs
+  ): Promise<GitOrganization> {
+    return await this.gitService.createGitOrganization(args);
+  }
+
+  @Mutation(() => Boolean)
+  @AuthorizeContext(
+    AuthorizableResourceParameter.GitOrganizationId,
+    'gitOrganizationId'
+  )
+  async deleteOrganization(@Args() args: BaseGitArgs): Promise<boolean> {
+    return this.gitService.deleteGitOrganization(args);
+  }
+
+  @Mutation(() => AuthorizeAppWithGithubResult)
+  //@AuthorizeContext(AuthorizableResourceParameter.WorkspaceId, 'workspaceId')
+  async getGithubAppInstallationUrl(
```

Comment:
This function should not get any parameters - since it is called "GitHub" it should use hardcoded value for "GitHub"